src/apic-extension/azext_apic_extension/aaz/latest/apic/api/definition/_export_specification.py

```diff
@@ -124,24 +123,10 @@ class ApiDefinitionsExportSpecification(AAZHttpOperation):
         def __call__(self, *args, **kwargs):
             request = self.make_request()
             session = self.client.send_request(request=request, stream=False, **kwargs)
-            if session.http_response.status_code in [202]:
-                return self.client.build_lro_polling(
-                    self.ctx.args.no_wait,
-                    session,
-                    self.on_200,
-                    self.on_error,
-                    lro_options={"final-state-via": "location"},
-                    path_format_arguments=self.url_parameters,
-                )
             if session.http_response.status_code in [200]:
-                return self.client.build_lro_polling(
```

Comment:
this is an async operation, should not be removed. Please check spec for async polling properties on location